features/features.go

```diff
@@ -85,9 +85,11 @@ var (
 					enableIn(configv1.Default, configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
 					mustRegister()
 
+	// OpenShift and Node Team will keep this turned off until evictions and
+	// disk provisioning are fixed even though upstream will take this GA.
```

Comment:
At a mechanical level, when a feature is GA, kube removes the gating logic from code.  Will we enforce this by preventing certain kubelet flags from being set?

Reply:
yes we'll turn it off by setting the KubeletConfiguration `NoSwap` value, and also keeping the `fail-swap-on` flag on. CNV will be able to turn on `LimitedSwap` and turn off `fail-swap-on`

Reply:
Yes, there will be something to block it, but the details have not been ironed out yet.

Reply:
we have plans to provide API level gating of that in MCO KubeletConfig controller (which hopefully will detect the presence of CNV to toggle the gating off, details still need to be figured out)